examples/environment-variables/package.json

```diff
@@ -7,7 +7,13 @@
   },
   "dependencies": {
     "next": "latest",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2"
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
+  },
+  "devDependencies": {
```

Comment:
Examples like this should add the `tsconfig.json` file by default. Could you address that?

Reply:
@balazsorban44 Sorry I don't follow, there is a `tsconfig.json` file present in all examples.

Reply:
You are correct, I was likely looking at the wrong branch. Thanks!